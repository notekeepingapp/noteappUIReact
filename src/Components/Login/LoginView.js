import React from 'react';

export const LoginView =(props)=>{
    return(
        <form onSubmit={props.validateUser}>
            <input
                value={props.username}
                name="username"
                type="text"
                onChange={event => props.handleUserInput(event)}
                style={{margin: "20px auto, display: block"}}
            />
            <input
                value={props.password}
                name="password"
                type="password"
                onChange={event => props.handleUserInput(event)}
                style={{margin: "20px auto, display: block"}}
            />
            <button
                type="submit"
                name="submit"
                style={{margin: "20px auto, display: block"}}
            >
                Submit
            </button>
        </form>
    )
};