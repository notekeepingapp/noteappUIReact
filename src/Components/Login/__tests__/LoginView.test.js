import React from 'react';
import { shallow } from 'enzyme';
import LoginView from '../LoginView';


describe('Login View rendering', () => {
  it('should match login view snapshot', () => {
    const component = shallow(<LoginView />);
    expect(component).toMatchSnapshot();
  });
});
