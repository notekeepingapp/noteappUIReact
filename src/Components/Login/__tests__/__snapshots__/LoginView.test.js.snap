// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login View rendering should match login view snapshot 1`] = `ShallowWrapper {}`;
